src/components/navigation/LoginCard.vue

```diff
@@ -1,52 +1,49 @@
 <template>
-  <v-card>
-    <v-card-title>Login</v-card-title>
-    <v-divider></v-divider>
-    <v-card-text>
-      <h4>Please select a realm to log in with</h4>
-    </v-card-text>
-    <v-card-actions>
-      <v-btn color="primary" @click="authEU" depressed>
-        EU
-      </v-btn>
-      <v-btn color="primary" @click="authNA" depressed>
-        NA
-      </v-btn>
-      <v-btn color="primary" @click="authASIA" depressed>
-        ASIA
-      </v-btn>
-      <v-btn color="primary" @click="authRU" depressed>
-        CIS
+  <v-card class="login-card">
+    <v-card-title class="login-card-title">
+      <div>{{ $t('navigation.login.card.title') }}</div>
+      <v-btn icon @click="closeHandler" v-if="isMobile">
+        <v-icon>mdi-close</v-icon>
       </v-btn>
-    </v-card-actions>
+    </v-card-title>
+    <v-card-text class="login-card-content">
+      <ProviderBlock v-for="(provider, name) in providers" :provider-name="name" :provider="provider" :key="name">
```

Comment:
Components should have hypen delimited versions of their imported name so for this it should be ```provider-block```. Vuetify writes it that way too fyi ;)